binding/rust/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "pv_porcupine"
+version = "0.1.0"
```

Comment:
version 1.9.0 (the last digit is your package version, the first two are the version of Porcupine)